examples/jsm/webxr/XRHandMeshModel.js

```diff
@@ -4,15 +4,20 @@ const DEFAULT_HAND_PROFILE_PATH = 'https://cdn.jsdelivr.net/npm/@webxr-input-pro
 
 class XRHandMeshModel {
 
-	constructor( handModel, controller, path, handedness ) {
+	constructor( handModel, controller, path, handedness, loader ) {
```

Comment:
To make this a bit more robust, can you please apply `null` as a default parameter? Further down, you can then write the if statement like so:
```js
if ( loader === null ) {
```

Reply:
Done!
Mind elaborating why that is more robust?

Reply:
We want to avoid generic checks like ` ! loader` and write them a bit more specific like `loader === null` or `loader === undefined`.